src/MainWindow.cpp

```diff
@@ -1809,7 +1809,7 @@ void MainWindow::onSynchronizationStarted()
 
 void MainWindow::onSynchronizationStopped()
 {
-    QNDEBUG(QStringLiteral("MainWindow::onSynchronizationStopped"));
+    QNWARNING(QStringLiteral("MainWindow::onSynchronizationStopped"));
```

Comment:
I don't think this particular slot is worthy of a warning: the appearance of a warning log entry is indicative of the fact something went wrong and most likely indicative of some bug. Users are free to stop the sync manually at their will, their explicit action should not grant a warning. I think `QNINFO` should be used here instead.

Reply:
Actually I did this message "Synchronization was stopped" often, when the sync was stopped by error, but OK I leave with INFO. We can fine tune later.